docs/content/en/docs-dev/user-guide/configuration-reference.md

```diff
@@ -516,6 +516,8 @@ One of `yamlField` or `regex` is required.
 
 There are some restrictions in configuring a service definition file.
 
+- As long as `desiredCount` is 0 or not set, `desiredCount` of your service will NOT be updated in deployments.
+  - If `desiredCount` is 0 or not set for a new service, the service's `desiredCount` will be 0.
 - `capacityProviderStrategy` is not supported.
```

Comment:
[Ask] When enabling ECS autoscaling, what should users do? In my understanding, they set `desiredCount` to 0 after setting up autoscaling. Is it right?

Reply:
Just to sure. I have the same concern.

Users can follow the below steps to create an autoscaled service. Is this correct?
1. create a service with the desiredCount as 0 or not specified. if it is not specified, the desiredCount is treated as 0.
1. configure autoscaling, and autoscaling policy or something like this updates desiredCount.
1. when updating a service, piped ignores the desiredCount. So the count of tasks is not touched.